.github/workflows/generate-pinval.yaml

```diff
@@ -149,57 +184,34 @@ jobs:
       - name: Install Hugo
         run: sudo dpkg -i ${{ runner.temp }}/hugo.deb
 
-      - name: Generate PINVAL reports (triad shard or explicit PINs)
-        run: |
-          set -euo pipefail
-
-          declare -a ARGS=(
-            --run-id "${{ inputs.run_id }}"
-            --triad "${{ inputs.triad_name }}"
-            --township "$TOWNSHIP"
-          )
-
-          # Convert comma-separated input into an array, stripping blanks
-          IFS=',' read -ra PINS_ARR <<< "${{ inputs.pins }}"
-          CLEAN_PINS=()
-          for p in "${PINS_ARR[@]:-}"; do
-            [[ -n "$p" ]] && CLEAN_PINS+=("$p")
-          done
```

Comment:
`read` should be able to handle removing extraneous whitespace, so I don't think we need this extra layer of processing anymore. Run this code for proof:

```
$ read -ra PINS_ARR <<< "123      456"; echo "${PINS_ARR[@]}"
123 456